bcipy/helpers/tests/test_convert.py

```diff
@@ -92,25 +97,63 @@ def test_convert_defaults(self):
 
         self.assertTrue(len(edf.get_data()) > 0)
 
-        for ch_name in ['c1', 'c2', 'c3']:
+        for ch_name in self.channels:
             self.assertTrue(ch_name in edf.ch_names)
 
     def test_overwrite_false(self):
         """Test overwriting fails"""
 
-        convert_to_edf(self.temp_dir)
+        convert_to_edf(self.temp_dir, mode=self.default_mode)
         with self.assertRaises(OSError):
-            convert_to_edf(self.temp_dir, overwrite=False)
+            convert_to_edf(self.temp_dir, mode=self.default_mode, overwrite=False)
 
     def test_overwrite_true(self):
         """Test that overwriting can be configured"""
 
-        convert_to_edf(self.temp_dir)
-        convert_to_edf(self.temp_dir, overwrite=True)
+        convert_to_edf(self.temp_dir, mode=self.default_mode)
+        convert_to_edf(self.temp_dir, overwrite=True, mode=self.default_mode)
 
     def test_with_custom_path(self):
         """Test creating the EDF without event annotations"""
         path = convert_to_edf(self.temp_dir,
+                              mode=self.default_mode,
                               edf_path=Path(self.temp_dir, 'mydata.edf'))
 
         self.assertEqual(Path(path).name, 'mydata.edf')
+
+    def test_with_write_targetness(self):
+        """Test creating the EDF without event annotations"""
+        path = convert_to_edf(self.temp_dir,
+                              mode=self.default_mode,
+                              write_targetness=True,
```

Comment:
If this is the default we probably want a separate test where `write_targetness=False` to check that the stim name is written correctly.